src/components/map/map.tsx

```diff
@@ -374,6 +379,27 @@ export class Map implements GxComponent {
     this.setMapProvider();
     this.map.setMaxZoom(this.maxZoom);
     this.fitBounds();
+
+    if (this.ShowMyLocation) {
+      this.map.locate({ enableHighAccuracy: true });
+      this.map.on("locationfound", e => {
+        console.log("Resultado location" + e.latlng.lat + "-" + e.latlng.lng);
+
+        LMarker(coords, {
+          icon: divIcon({
+            className: this.markerClassIcon,
+            iconAnchor: [40, 30],
+            popupAnchor: [0, 10],
+            iconSize: [40, 40],
+            tooltipAnchor: [0, 10]
+          })
+        })
+          .bindPopup("<i>Ubicacion Actual! </i>")
```

Comment:
Avoid fixed message strings in your code. More even, if they are in spanish. 
If the string can't be bound to a `gx-map`'s property, at least it will be fine if the string is in English.

The best case scenario is to bind this message to a property that can be translated.